devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -12,15 +12,19 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Z-Wave Dimmer Switch Generic", namespace: "smartthings", author: "SmartThings", category: "C5") {
 		capability "Switch Level"
 		capability "Actuator"
 		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
+		capability "Configuration"
+		capability "Health Check"
 
 		fingerprint inClusters: "0x26", deviceJoinName: "Z-Wave Dimmer"
+		fingerprint mfr:"0063", prod:"4944", deviceJoinName: "Z-Wave In-Wall Dimmer"
```

Comment:
@pchomal  - We confirmed. The fingerprints you added here is already covered in `dimmer-switch.groovy` DTH. Please `Health Check` that DTH and take out these fingerprints.